syntax = "proto2";

message RedeemSwap {
    required bytes swapID = 1;
    }

